### DISCUSSION TITLE
Still I have not received education benefit.

### DISCUSSION THREAD

**Discussion by Sakib99-cyber** [community/community#141475] [GitHub Education] [answered]
### Hi everyone,

It's been almost 3 days, my application has been approved. But still I have not received education benefit. How many days can it take to get it? Or what do I have to do? What is the solution? How long to wait! Can anyone help me!
User: Sakib99-cyber

![Screenshot (28)](https://github.com/user-attachments/assets/52eaf5c8-a21f-47c5-b93a-0e88529d0685)

**Comment by davevad93** [comment]
Hi @Sakib99-cyber ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Sakib99-cyber was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Sakib99-cyber** [reply]
**Waiting. . .**

**Reply by davevad93** [reply, marked as answer]
Don't worry @Sakib99-cyber you'll get your benefits soon! 👍

**Reply by Sakib99-cyber** [reply]
Finally got it . Thank you @davevad93

**Reply by davevad93** [reply]
You're welcome @Sakib99-cyber 👍
